### ISSUE TITLE
Печать копии чека

### ISSUE THREAD

**Issue by susloparov77** [shtrih-m/fr_drv_ng#25]
Здравствуйте, выполняю код, пишет ошибку:

uint32_t adminPassword = 30;
ci.Set_DocumentNumber(88);
ci.Set_ShowTagNumber(false);
ci.Set_Password(adminPassword);
ci.FNGetDocumentAsString();

std::string str = ci.Get_StringForPrinting();
std::size_t already_print = 0;
std::string str2;
std::size_t eos;

while((eos = str.find('\n', already_print)) != std::string::npos)
{
   str2 = str.substr(already_print, eos - already_print);
   already_print = eos + 1;
   ci.Set_StringForPrinting(str2);
   ci.PrintString();
}

str2 = str.substr(already_print);
if (!str2.empty()){
    ci.Set_StringForPrinting(str2);
    ci.PrintString();
}

Я правильно же передаю номер документа в фн??
Логи прилагаю.

[fr_drv.log](https://github.com/shtrih-m/fr_drv_ng/files/2610977/fr_drv.log)

**Comment by swex** [comment]
Всё правильно, значит нет такого документа в ФН.
Помните что всегда нужно проверять код возврата от методов.
Попробуйте получить  последний документ сразу после операции
Вот так:
```c++
    checkResult(ci.FNGetStatus()); //DocumentNumber становится последним документом в ФН
    checkResult(ci.FNGetDocumentAsString()); //получаем его себе как строку
```

**Comment by susloparov77** [comment]
А как нету если у меня есть чек и нам написано номер в ФД:88 ? Какой номер вводить тогда???

**Comment by susloparov77** [comment]
Скажите плиз какой номер вводить??? Он как называется на чеке??

**Comment by swex** [comment]
у Вас в коде всё ок. У нас там был баг, который проявлялся только на КЯ.  Уже поправили, скоро будет сборка.

**Comment by susloparov77** [comment]
Скажите плиз когда скачать, достаточно заменить ваши библиотеки *.so?

**Comment by swex** [comment]
да, если не менялась мажорная/минорная версия (первая и вторая цифра в версии).
Выложена обновленная сборка
